client/my-sites/google-my-business/select-business-type/index.js

```diff
@@ -0,0 +1,92 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import HeaderCake from 'components/header-cake';
+import Card from 'components/card';
+import CTACard from './cta-card';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+class SelectBusinessType extends Component {
+	static propTypes = {
+		translate: PropTypes.func.isRequired,
+		siteId: PropTypes.string.isRequired,
+		recordTracksEvent: PropTypes.func.isRequired,
+	};
+
+	onCreateMyListingClick = () => {
+		this.props.recordTracksEvent( 'calypso_google_my_business_create_my_listing_click' );
```

Comment:
We've been prefixing new events with `test` in the nudge, let's follow this pattern until we're ready to make this new feature public. I also suspect the suffix should be `clicked` instead of `click`. Same thing for the other event below.

Reply:
This has not been addressed.